docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises-manual.md

```diff
@@ -295,149 +298,27 @@ On each node, perform the following as a user with sudo access:
     sudo systemctl status node_exporter
     ```
 
-## Install backup utilities
-
-YugabyteDB Anywhere supports backing up YugabyteDB to Amazon S3, Azure Storage, Google Cloud Storage, and Network File System (NFS). For more information, see [Configure backup storage](../../../back-up-restore-universes/configure-backup-storage/).
-
-You can install the backup utility for the backup storage you plan to use, as follows:
-
-- NFS: Install rsync, which YugabyteDB Anywhere uses to perform NFS backups installed during one of the previous steps.
```

Comment:
I think rsync is still required even with YBC?

Reply:
Spoke with @Vars-07 and it is not required, so we can safely remove this too. Should we also remove that from our Ansible playbooks too then?

Reply:
Makes sense. One scenario I can think off is when someone wants to restore the non ybc based backups - then that will be required. Though it's a rare scenario but can exists.
cc: @kv83821-yb for confirmation as well